JavaScript/script.js

```diff
@@ -45,4 +45,19 @@ function openLab(event) {
             tab.style.display = 'none';
         }
     })
+}
+
+
+//collaspible menu animation
+var coll = document.getElementsByClassName("collapsible");
+for(var i=0; i < coll.length; i++){
+    coll[i].addEventListener("click", function(){
+        this.classList.toggle("active");
+        var content = this.nextElementSibling;
+        if(content.style.maxHeight){
+            content.style.maxHeight = null;
+        } else{
+            content.style.maxHeight = content.scrollHeight + "px";
+        }
+    });
```

Comment:
- use the `let` keyword instead of `var` to declare variables
- ES6 JavaScript can use a forEach loop instead of a vanilla for loop, look into that. Also look at template strings. For example:
```
let i = "string";
console.log(`${i}`) prints string
```